openmetadata-ui/src/main/resources/ui/src/utils/BrowserNotificationUtils.ts

```diff
@@ -11,8 +11,11 @@
  *  limitations under the License.
  */
 
+export const isNotificationAPIAvailable = () => 'Notification' in window;
```

Comment:
Can we store it as variable and not as function?

Reply:
We can do that, but I think storing it in function will be more appropriate if we want to check it multiple times throughout our code.

Reply:
I will change it to variable since we do not want to check it multiple times.